tensorflow/python/ops/rnn.py

```diff
@@ -515,7 +515,7 @@ def dynamic_rnn(cell, inputs, sequence_length=None, initial_state=None,
       state = initial_state
     else:
       if not dtype:
-        raise ValueError("If no initial_state is provided, dtype must be.")
+        raise ValueError("A dtype must be specified if no initial_state is provided.")
```

Comment:
style nit: This line is now > 80 chars long. Can you please break the line at the first double quote and use a hanging indent of four space related to the previous line.